consensus/consensus-types/src/block.rs

```diff
@@ -319,7 +319,13 @@ impl<T> From<&Block<T>> for BlockMetadata {
             block.id(),
             block.round(),
             block.timestamp_usecs(),
-            block.quorum_cert().ledger_info().signatures().clone(),
+            block
+                .quorum_cert()
+                .ledger_info()
+                .signatures()
+                .keys()
```

Comment:
Is `keys` deterministic? Guessing so, but might be useful to emphasize in a comment too.

Reply:
it is because it's from a BTreeMap but sure a comment can help